README.md

```diff
@@ -97,7 +169,7 @@ or specific version:
 
 ## Kotlin, Kotlin Jupyter, OpenAPI, Arrow and JDK versions
 
-This table shows the mapping between main library components versions and minimum supported Java versions.
+This table shows the mapping between main library component versions and minimum supported Java versions.
 
 | Kotlin DataFrame Version | Minimum Java Version | Kotlin Version | Kotlin Jupyter Version | OpenAPI version | Apache Arrow version |
```

Comment:
make sure to rebase on the master branch, so these versions are correct again :)